### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: docs/CODE_QUALITY.md [hunk 1]
```diff
@@ -69,16 +69,12 @@ available functionality:
 13. When building systemd from a git checkout the build scripts will
     automatically enable a git commit hook that ensures whitespace cleanliness.
 
-14. [LGTM](https://lgtm.com/) analyzes every commit pushed to `main`. The list
-    of active alerts can be found
-    [here](https://lgtm.com/projects/g/systemd/systemd/alerts/?mode=list).
```
Comment (yuwata) [inline docs/CODE_QUALITY.md:74]: @mrc0mmand Are there any good alternative pages for CodeQL?

**Review by DaanDeMeyer — APPROVED** [review]

**Comment by poettering** [comment]
@keszybz, those graphs are dead and gone from your servers? :-(

File: README.md [hunk 2]
```diff
@@ -2,14 +2,11 @@
...
-<a href="http://in.waw.pl/systemd-github-state/systemd-systemd-pull-requests.svg"><img align="right" src="http://in.waw.pl/systemd-github-state/systemd-systemd-pull-requests-small.svg" alt="Count of open pull requests over time"></a>
 [![Semaphore CI 2.0 Build Status](https://the-real-systemd.semaphoreci.com/badges/systemd/branches/main.svg?style=shields)](https://the-real-systemd.semaphoreci.com/projects/systemd)<br/>
 [![Coverity Scan Status](https://scan.coverity.com/projects/350/badge.svg)](https://scan.coverity.com/projects/350)<br/>
 [![OSS-Fuzz Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/systemd.svg)](https://oss-fuzz-build-logs.storage.googleapis.com/index.html#systemd)<br/>
 [![CIFuzz](https://github.com/systemd/systemd/workflows/CIFuzz/badge.svg)](https://github.com/systemd/systemd/actions)<br/>
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1369/badge)](https://bestpractices.coreinfrastructure.org/projects/1369)<br/>
-[![Language Grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/systemd/systemd.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/systemd/systemd/context:cpp)<br/>
```
Comment (mrc0mmand) [inline README.md]: I'm surprised this doesn't cause a conflict with https://github.com/systemd/systemd/commit/c03896235dc840c902a2f02d9c8a106ff19de2bd :-)

**Review by mrc0mmand — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (yuwata) [reply README.md]: Interesting. I did not notice you already dropped the line.

**Comment by yuwata** [comment]
Rebased, to 'resolve' the 'conflict'.

**Comment by keszybz** [comment]
> @keszybz, those graphs are dead and gone from your servers? :-(

Yeah, I haven't had any spare time to bring them back. If that ever happens, we can easily re-add the link.

**Pull request merged**